.github/workflows/build.yml

```diff
@@ -41,11 +43,11 @@ jobs:
       shell: bash -xe {0}
       run: |
         declare -A versions=(
-          ["master"]="15.1"
-          ["REL_15_STABLE"]="15.1"
-          ["REL_14_STABLE"]="14.6"
-          ["REL_13_STABLE"]="13.9"
-          ["REL_12_STABLE"]="12.13"
+          ["master"]="16.1"
+          ["REL_15_STABLE"]="15.5"
```

Comment:
Do you need to add REL_16_STABLE?

Reply:
Looking at last year's history, I will create REL_16_STABLE after copyright and other modifications.